### PR TITLE
Polishing Dashboard page

### PR DESCRIPTION
**Pull request by pva701**
## Related issue(s)

https://issues.serokell.io/issue/EDNA-80
https://issues.serokell.io/issue/EDNA-88

#### Related changes (conditional)

- Tests
  - [ ] If I added new functionality, I added tests covering it.
  - [ ] If I fixed a bug, I added a regression test to prevent the bug from
        silently reappearing again.

- Documentation
  - [ ] I checked whether I should update the docs and did so if necessary:
    - [README](../tree/master/README.md)

#### Stylistic guide (mandatory)

- [ ] My commits comply with [the policy used in Serokell](https://www.notion.so/serokell/Commit-and-PR-policy-4cf98e1a910a415d86b5f2491d9af1af).
- [ ] My code complies with the [style guide](../tree/master/docs/code-style.md).

### CODE REVIEW

File: frontend/src/pages/dashboard/ExperimentsTable/ExperimentsTable.tsx [hunk 1]
```diff
@@ -126,10 +139,41 @@ export function ExperimentsTableSuspendable({
...
+          <ContextActions
+            actions={[
+              <div
+                key="metadata"
+                className="contextActions__item"
+                onMouseDown={async () => {
+                  const metadata = await cacheMetadata(e.id);
+                  setModalDialog({
+                    kind: "show-experiment-metadata",
+                    description: metadata.fileMetadata.concat(metadata.description),
```
Comment (gromakovsky) [inline frontend/src/pages/dashboard/ExperimentsTable/ExperimentsTable.tsx]: I see that now description is shown last, i. e. after all other metadata. I think description is likely to be more important than `fileMetadata` and it's better to show it first, but I am not entirely sure so it's up to you to decide.

**Comment by gromakovsky** [comment]
Found something weird. I don't know what's the source of this bug, but maybe it's introduced in this PR, in this case please fix it.

image

**Comment by gromakovsky** [comment]
According to https://serokell.slack.com/archives/C01MH299XEU/p1618811108000800, names of primary sub-experiments should be in **bold**. Can you please implement it here?

**Review by gromakovsky — APPROVED** [review]
My concerns seem to be fixed.

**Comment by sashasashasasha151** [comment]
image
Don't see error on invalid sub-experiments

**Comment by sashasashasasha151** [comment]
image
Plot is not in full height

**Comment by sashasashasasha151** [comment]
image
No margin in the bottom

File: frontend/src/components/DialogLayout/DialogLayout.scss [hunk 2]
```diff
@@ -77,6 +77,11 @@
 .secondaryDialogWindow {
   border-radius: 6 * $px;
 
+  p {
+    margin-top: 2 * $px;
+    margin-bottom: 2 * $px;
```
Comment (sashasashasasha151) [inline frontend/src/components/DialogLayout/DialogLayout.scss:82]: Still don't understand why you like splitting it

**Comment by gromakovsky** [comment]
I'll create a few issues to follow-up. I want to merge it now because we are going to have a demo today.

**Comment by gromakovsky** [comment]
New issues regarding dashboard frontend:
* https://issues.serokell.io/issue/EDNA-99
* https://issues.serokell.io/issue/EDNA-100
* https://issues.serokell.io/issue/EDNA-102

**Pull request merged**